src/libraries/System.Data.Common/src/System/Data/Common/StringStorage.cs

```diff
@@ -57,7 +57,7 @@ public override object Aggregate(int[] recordNos, AggregateType kind)
                     }
                     if (max >= 0)
                     {
-                        for (i = i + 1; i < recordNos.Length; i++)
+                        for (++i; i < recordNos.Length; i++)
```

Comment:
Why did this one become `++i` and not `i++`?

Reply:
I'm not sure what the fixers rules are here.
```suggestion
                        for (i++; i < recordNos.Length; i++)
```